routes/auth.js

```diff
@@ -93,4 +93,34 @@ router.post("/register",
         }
 })
 
+
+router.get('/me', isAuthenticated, async (req, res) => { 
+
+    const { userId } = req.user.token
+
+    try {
+        const data = await User.findByPk(userId)        
+        if(data.dataValues !== undefined && data.dataValues !== null) {
+            const { firstName, lastName, email } = data.dataValues
+            res.status(203).json({
+                message: "Datos del user",
+                data: {
+                    firstName,
+                    lastName, 
+                    email
+                } 
+            })    
+        } else {
+            res.status(204).json({
```

Comment:
>[Express response.status(204) Won’t Return Content](https://bambielli.com/til/2017-03-26-express-res-204/)
[...]express response objects will not forward a response body if the response status code is 204 No Content.[...]